native/acter/Cargo.toml

```
@@ -42,17 +42,17 @@ mime2ext = "0.1.52"
 mime_guess = "2.0.4"
 parse-env-filter = "0.1.0"
 ruma = { workspace = true, features = ["html", "rand", "unstable-msc2448", "unstable-msc2965", "unstable-msc3930", "unstable-msc3245-v1-compat"] }
+ruma-client-api = { workspace = true }
 ruma-common = { workspace = true, features = ["client", "unstable-unspecified"] }
 ruma-events = { workspace = true }
 sanitize-filename-reader-friendly = "2.2.1"
 serde = { version = "1", features = ["derive"] }
-serde_json = "1.0.111"
+serde_json = "1"
 strum = { workspace = true }
-tokio = "1.7"
+tokio = "1"
```

```suggestion
tokio = "1.7"
```

we need to keep this at the 1.7 minimum.

---

Fixed by https://github.com/acterglobal/a3/pull/1346/commits/cfb67aaf65bfe250bf39a8016ca53eb22c8f3e51